docs/content/en/docs/tutorials/_index.md

```diff
@@ -26,3 +26,4 @@ Take a look at our other guides:
 | Detailed Tutorials |
 |--------------------|
 | [Custom Build Script]({{< relref "/docs/tutorials/custom-builder" >}}) |
+| [Inter-Artifact Dependencies]({{< relref "/docs/tutorials/artifact-dependencies" >}}) |
```

Comment:
I dont like the sound of `Inter-Artifact`. should we just call is artifact dependency?
```suggestion
| [Artifact Dependencies]({{< relref "/docs/tutorials/artifact-dependencies" >}}) |
```

Reply:
done